### ISSUE TITLE
Lighten requirements?

### ISSUE THREAD

**Issue by zmoon** [jbusecke/xmovie#71]
`dask` and `cartopy` could be optional requirements, with exceptions raised if code that needs them is called without them installed, as they are not needed for the core functionality

**Comment by jbusecke** [comment]
This seems very reasonable indeed. I currently do not have the bandwidth to implement this though. Do you think you would have time to submit a PR? No pressure at all, though. Raising this issue is already very helpful.

**Comment by zmoon** [comment]
Yes I will give it a try after #67

**Comment by jbusecke** [comment]
Excellent. I really appreciate your efforts here @zmoon!
